From: Wang Shilong <wangsl.fnst@cn.fujitsu.com>
Date: Fri, 25 Oct 2013 18:52:08 +0800
Patch-mainline: 3.13
Git-commit: 7fdf4b608dda5eea114cb23623b52e34dd5972f5
Subject: [PATCH] Btrfs: use 'u64' rather than 'int' to get extent's
 generation

We define a 'int' to get extent's generation by mistake,fix it.

Signed-off-by: Wang Shilong <wangsl.fnst@cn.fujitsu.com>
Signed-off-by: Josef Bacik <jbacik@fusionio.com>
Signed-off-by: Chris Mason <chris.mason@fusionio.com>
Signed-off-by: David Sterba <dsterba@suse.cz>
---
 fs/btrfs/relocation.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/fs/btrfs/relocation.c
+++ b/fs/btrfs/relocation.c
@@ -3263,7 +3263,7 @@ static int add_tree_block(struct reloc_c
 	struct rb_node *rb_node;
 	u32 item_size;
 	int level = -1;
-	int generation;
+	u64 generation;
 
 	eb =  path->nodes[0];
 	item_size = btrfs_item_size_nr(eb, path->slots[0]);
